Branch about us

YangYue128-helen opened the pull request:
Update AboutUs

YangYue128-helen committed:
Update AboutUs

Avalionnet commented on docs/AboutUs.md:
```diff
@@ -38,15 +38,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-[[github](http://github.com/johndoe)]
+[[github](https://github.com/YangYue128-helen)]
 [[portfolio](team/johndoe.md)]
```
Hmm maybe you can consider updating this too as they will probably ask us to do it in the weeks ahead. But it's not essential for this iteration so no worries!

Avalionnet reviewed (approved):
LGTM!! Do check out the comment too!

YangYue128-helen committed:
Update portfolio

YangYue128-helen committed:
Resolve conflicts on AboutUs
Merge branch 'YangYue128-helen-branch-AboutUs'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

YangYue128-helen committed:
Resolve image location

JinHao-L reviewed (approved):
LGTM! 👍

WadeForward reviewed (commented):
LGTM! 👍
